File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int len_s = s.length();
        int len_t = t.length();
        if(len_s != len_t){
            return false;
        }
        int count_array[26] = {0};
        for(int i=0; i<len_s; i++){
            count_array[s[i]-'a'] += 1;
            count_array[t[i]-'a'] -= 1;
        }
        for(int i=0; i<26; i++){
            if(count_array[i]){
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
from collections import Counter
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if(len(s)!=len(t)):
            return False

        counter_s = Counter(s)
        for char in t:
            counter_s[char] -= 1
            
        for key in counter_s.keys():
            if (counter_s[key]!=0):
                return False
        
        return True